Vlv2TbltDevicePkg/SampleApplication/SampleApplication.c

```diff
@@ -0,0 +1,67 @@
+/** @file
+  Sample UEFI Application 
+**/
```

Comment:
@krystian-hebel we need correct copyrights.

Reply:
done